tests/unit_tests/test_helm_chart.py

```diff
@@ -318,6 +318,35 @@ def test_worker_scratch_config_used_when_init_container_enabled():
     assert config.scratch == init_config.scratch
 
 
+@pytest.mark.parametrize("enabled", [True, False])
+def test_fluentd_ignore_true_when_graylog_enabled(enabled):
+    manifests = render_chart(
+        values={
+            "worker": {
+                "logging": {
+                    "graylog_enabled": enabled,
+                },
+            },
+        }
+    )
+
+    if enabled:
```

Comment:
Nit: I would personally write this as two tests, it's roughly the same number of lines of code and much easier to read, you can also extract the common lines into their own function.